Formula/simg2img.rb

```diff
@@ -0,0 +1,26 @@
+class Simg2img < Formula
+  desc "Tool to convert Android sparse images to raw images and back"
+  homepage "https://github.com/anestisb/android-simg2img"
+  url "https://github.com/anestisb/android-simg2img/archive/1.0.0.tar.gz"
+  sha256 "5e93661135cad56117db5523a3bb0edd73fe9e31802bbfa3b1f165bc89fdb3ef"
+
+  head do
```

Comment:
Don't need this block. Just `head ""https://github.com/anestisb/android-simg2img.git"` under the `sha256` line above.